### ISSUE TITLE
[chdbits] (updating) FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Error: Unable to process browser request. TimeoutError: Navigation timeout of 25000 ms exceeded

### ISSUE THREAD

**Issue by hoong7** [FlareSolverr/FlareSolverr#1210]
### Have you checked our README?

- [X] I have checked the README

### Have you followed our Troubleshooting?

- [X] I have followed your Troubleshooting

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version:
- Last working FlareSolverr version:
- Operating system:
- Are you using Docker: [yes/no]
- FlareSolverr User-Agent (see log traces or / endpoint):
- Are you using a VPN: [yes/no]
- Are you using a Proxy: [yes/no]
- Are you using Captcha Solver: [yes/no]
- If using captcha solver, which one:
- URL to test this issue:
```

### Description

Unable to add https://ptchdbits.co/index.php

### Logged Error Messages

```text
Flaresolverr
date	stream	content
2024/06/10 17:32:48	stdout	2024-06-10T09:32:48+00:00 INFO REQ-16 Response in 26.333 s
2024/06/10 17:32:48	stdout	2024-06-10T09:32:48+00:00 ERROR REQ-16 Error: Unable to process browser request. TimeoutError: Navigation timeout of 25000 ms exceeded
2024/06/10 17:32:48	stdout	2024-06-10T09:32:48+00:00 ERROR REQ-16 Unexpected error: TimeoutError: Navigation timeout of 25000 ms exceeded
2024/06/10 17:32:21	stdout	2024-06-10T09:32:21+00:00 INFO REQ-16 Incoming request => POST /v1 body: {"maxTimeout":50000,"cmd":"request.get","url":"https://ptchdbits.co/login.php"}
2024/06/10 17:31:21	stdout	2024-06-10T09:31:21+00:00 INFO REQ-15 Response in 26.482 s
2024/06/10 17:31:21	stdout	2024-06-10T09:31:21+00:00 ERROR REQ-15 Error: Unable to process browser request. TimeoutError: Navigation timeout of 25000 ms exceeded
2024/06/10 17:31:21	stdout	2024-06-10T09:31:21+00:00 ERROR REQ-15 Unexpected error: TimeoutError: Navigation timeout of 25000 ms exceeded
2024/06/10 17:30:55	stdout	2024-06-10T09:30:55+00:00 INFO REQ-15 Incoming request => POST /v1 body: {"maxTimeout":50000,"cmd":"request.get","url":"https://ptchdbits.co/login.php"}
2024/06/10 17:30:34	stdout	2024-06-10T09:30:34+00:00 INFO REQ-14 Incoming request => GET /
2024/06/10 17:29:27	stdout	2024-06-10T09:29:27+00:00 INFO REQ-13 Response in 11.197 s
2024/06/10 17:29:27	stdout	2024-06-10T09:29:27+00:00 ERROR REQ-13 Error: Unable to process browser request. TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:29:27	stdout	2024-06-10T09:29:27+00:00 ERROR REQ-13 Unexpected error: TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:29:16	stdout	2024-06-10T09:29:16+00:00 INFO REQ-13 Incoming request => POST /v1 body: {"maxTimeout":20000,"cmd":"request.get","url":"https://ptchdbits.co/login.php"}
2024/06/10 17:29:13	stdout	2024-06-10T09:29:13+00:00 INFO REQ-12 Response in 11.217 s
2024/06/10 17:29:13	stdout	2024-06-10T09:29:13+00:00 ERROR REQ-12 Error: Unable to process browser request. TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:29:13	stdout	2024-06-10T09:29:13+00:00 ERROR REQ-12 Unexpected error: TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:29:02	stdout	2024-06-10T09:29:02+00:00 INFO REQ-12 Incoming request => POST /v1 body: {"maxTimeout":20000,"cmd":"request.get","url":"https://ptchdbits.co/login.php"}
2024/06/10 17:28:57	stdout	2024-06-10T09:28:57+00:00 INFO REQ-11 Response in 11.207 s
2024/06/10 17:28:57	stdout	2024-06-10T09:28:57+00:00 ERROR REQ-11 Error: Unable to process browser request. TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:28:57	stdout	2024-06-10T09:28:57+00:00 ERROR REQ-11 Unexpected error: TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:28:45	stdout	2024-06-10T09:28:45+00:00 INFO REQ-11 Incoming request => POST /v1 body: {"maxTimeout":20000,"cmd":"request.get","url":"https://ptchdbits.co/login.php"}
2024/06/10 17:28:45	stdout	2024-06-10T09:28:45+00:00 INFO REQ-10 Response in 11.155 s
2024/06/10 17:28:45	stdout	2024-06-10T09:28:45+00:00 ERROR REQ-10 Error: Unable to process browser request. TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:28:45	stdout	2024-06-10T09:28:45+00:00 ERROR REQ-10 Unexpected error: TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:28:34	stdout	2024-06-10T09:28:34+00:00 INFO REQ-10 Incoming request => POST /v1 body: {"maxTimeout":20000,"cmd":"request.get","url":"https://ptchdbits.co/login.php"}
2024/06/10 17:28:33	stdout	2024-06-10T09:28:33+00:00 INFO REQ-9 Response in 13.189 s
2024/06/10 17:28:33	stdout	2024-06-10T09:28:33+00:00 ERROR REQ-9 Error: Unable to process browser request. TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:28:33	stdout	2024-06-10T09:28:33+00:00 ERROR REQ-9 Unexpected error: TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:28:20	stdout	2024-06-10T09:28:20+00:00 INFO REQ-9 Incoming request => POST /v1 body: {"maxTimeout":20000,"cmd":"request.get","url":"https://ptchdbits.co/login.php"}
2024/06/10 17:20:12	stdout	2024-06-10T09:20:12+00:00 INFO REQ-8 Response in 11.209 s
2024/06/10 17:20:12	stdout	2024-06-10T09:20:12+00:00 ERROR REQ-8 Error: Unable to process browser request. TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:20:12	stdout	2024-06-10T09:20:12+00:00 ERROR REQ-8 Unexpected error: TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:20:01	stdout	2024-06-10T09:20:01+00:00 INFO REQ-8 Incoming request => POST /v1 body: {"maxTimeout":20000,"cmd":"request.get","url":"https://pt.btschool.club/login.php"}
2024/06/10 17:19:53	stdout	2024-06-10T09:19:53+00:00 INFO REQ-7 Response in 11.823 s
2024/06/10 17:19:53	stdout	2024-06-10T09:19:53+00:00 ERROR REQ-7 Error: Unable to process browser request. TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:19:53	stdout	2024-06-10T09:19:53+00:00 ERROR REQ-7 Unexpected error: TimeoutError: Navigation timeout of 10000 ms exceeded
2024/06/10 17:19:42	stdout	2024-06-10T09:19:42+00:00 INFO REQ-7 Incoming request => POST /v1 body: {"maxTimeout":20000,"cmd":"request.get","url":"https://pt.btschool.club/login.php"}
2024/06/10 17:18:48	stdout	2024-06-10T09:18:48+00:00 INFO REQ-6 Response in 28.804 s
2024/06/10 17:18:48	stdout	2024-06-10T09:18:48+00:00 ERROR REQ-6 Error: Unable to process browser request. TimeoutError: Navigation timeout of 27500 ms exceeded
2024/06/10 17:18:48	stdout	2024-06-10T09:18:48+00:00 ERROR REQ-6 Unexpected error: TimeoutError: Navigation timeout of 27500 ms exceeded
2024/06/10 17:18:19	stdout	2024-06-10T09:18:19+00:00 INFO REQ-6 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://pt.btschool.club/login.php"}
2024/06/10 17:18:19	stdout	2024-06-10T09:18:19+00:00 INFO REQ-5 Response in 28.673 s
2024/06/10 17:18:19	stdout	2024-06-10T09:18:19+00:00 ERROR REQ-5 Error: Unable to process browser request. TimeoutError: Navigation timeout of 27500 ms exceeded
2024/06/10 17:18:19	stdout	2024-06-10T09:18:19+00:00 ERROR REQ-5 Unexpected error: TimeoutError: Navigation timeout of 27500 ms exceeded
2024/06/10 17:17:50	stdout	2024-06-10T09:17:50+00:00 INFO REQ-5 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://pt.btschool.club/login.php"}
2024/06/10 17:17:50	stdout	2024-06-10T09:17:50+00:00 INFO REQ-4 Response in 28.823 s
2024/06/10 17:17:50	stdout	2024-06-10T09:17:50+00:00 ERROR REQ-4 Error: Unable to process browser request. TimeoutError: Navigation timeout of 27500 ms exceeded
2024/06/10 17:17:50	stdout	2024-06-10T09:17:50+00:00 ERROR REQ-4 Unexpected error: TimeoutError: Navigation timeout of 27500 ms exceeded
2024/06/10 17:17:22	stdout	2024-06-10T09:17:22+00:00 INFO REQ-4 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://pt.btschool.club/login.php"}
2024/06/10 17:17:21	stdout	2024-06-10T09:17:21+00:00 INFO REQ-3 Response in 31.691 s
2024/06/10 17:17:21	stdout	2024-06-10T09:17:21+00:00 ERROR REQ-3 Error: Unable to process browser request. TimeoutError: Navigation timeout of 27500 ms exceeded
2024/06/10 17:17:21	stdout	2024-06-10T09:17:21+00:00 ERROR REQ-3 Unexpected error: TimeoutError: Navigation timeout of 27500 ms exceeded
2024/06/10 17:16:50	stdout	2024-06-10T09:16:50+00:00 INFO REQ-3 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://pt.btschool.club/login.php"}
2024/06/10 16:24:38	stdout	2024-06-10T08:24:38+00:00 INFO REQ-2 Incoming request => GET /
2024/06/10 13:42:43	stdout	2024-06-10T05:42:43+00:00 INFO REQ-1 Incoming request => GET /
2024/06/10 13:42:26	stdout	2024-06-10T05:42:26+00:00 INFO Listening on http://0.0.0.0:8191
2024/06/10 13:42:26	stdout	2024-06-10T05:42:26+00:00 INFO Test successful
2024/06/10 13:42:26	stdout	2024-06-10T05:42:26+00:00 INFO FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:89.0) Gecko/20100101 Firefox/89.0
2024/06/10 13:42:21	stdout	2024-06-10T05:42:21+00:00 INFO Testing web browser installation...
2024/06/10 13:42:21	stdout	2024-06-10T05:42:21+00:00 INFO FlareSolverr v2.1.0
```

### Screenshots

![微信截图_20240610173458](https://github.com/FlareSolverr/FlareSolverr/assets/71035460/e9db26cd-4743-4443-b7f4-8821d3084a4c)

**Comment by ilike2burnthing** [comment]
Your FlareSolverr version is 2.5 years out of date. Update to the latest (v3.3.19) and use at least the default timeout, if not longer.

If you still have issues, reopen with environment information as initially requested.

**Comment by hoong7** [comment]
![123](https://github.com/FlareSolverr/FlareSolverr/assets/71035460/b8655328-4e7f-4ec5-b134-785150ff92e1)



<html>
<body>
<h2 style="text-align: center; color: rgb(0, 0, 0); font-family: &quot;Lucida Console&quot;, Consola, Monaco, monospace, &quot;Courier New&quot;, Courier, serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Flaresolverr</h2>
date | stream | content
-- | -- | --
2024/06/10 22:42:45 | stdout | 2024-06-10 14:42:45 INFO     172.17.0.1 POST http://10.123.45.199:8191/v1 500 Internal Server Error
2024/06/10 22:42:45 | stdout | 2024-06-10 14:42:45 INFO     Response in 50.603 s
2024/06/10 22:42:45 | stdout | 2024-06-10 14:42:45 ERROR    Error: Error solving the challenge. Timeout after 50.0 seconds.
2024/06/10 22:41:58 | stdout | 2024-06-10 14:41:58 INFO     Challenge detected. Selector found: #challenge-spinner
2024/06/10 22:41:54 | stdout | version_main cannot be converted to an integer
2024/06/10 22:41:54 | stdout | 2024-06-10 14:41:54 INFO     Incoming request => POST /v1 body: {'maxTimeout': 50000, 'cmd': 'request.get', 'url': 'https://ptchdbits.co/login.php'}
2024/06/10 22:33:36 | stdout | 2024-06-10 14:33:36 INFO     172.17.0.1 POST http://10.123.45.199:8191/v1 500 Internal Server Error
2024/06/10 22:33:36 | stdout | 2024-06-10 14:33:36 INFO     Response in 50.686 s
2024/06/10 22:33:36 | stdout | 2024-06-10 14:33:36 ERROR    Error: Error solving the challenge. Timeout after 50.0 seconds.
2024/06/10 22:32:50 | stdout | 2024-06-10 14:32:50 INFO     Challenge detected. Selector found: #challenge-spinner
2024/06/10 22:32:46 | stdout | version_main cannot be converted to an integer
2024/06/10 22:32:46 | stdout | 2024-06-10 14:32:46 INFO     Incoming request => POST /v1 body: {'maxTimeout': 50000, 'cmd': 'request.get', 'url': 'https://ptchdbits.co/login.php'}
2024/06/10 22:32:02 | stdout | 2024-06-10 14:32:02 INFO     172.17.0.1 GET http://10.123.45.199:8191/ 200 OK
2024/06/10 22:30:46 | stdout | 2024-06-10 14:30:46 INFO     Serving on http://0.0.0.0:8191
2024/06/10 22:30:46 | stdout | 2024-06-10 14:30:46 INFO     Test successful!
2024/06/10 22:30:46 | stdout | 2024-06-10 14:30:46 INFO     FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36
2024/06/10 22:30:45 | stdout | version_main cannot be converted to an integer
2024/06/10 22:30:45 | stdout | 2024-06-10 14:30:45 INFO     Launching web browser...
2024/06/10 22:30:45 | stdout | 2024-06-10 14:30:45 INFO     Chrome / Chromium major version: 120
2024/06/10 22:30:45 | stdout | 2024-06-10 14:30:45 INFO     Chrome / Chromium path: /usr/bin/chromium
2024/06/10 22:30:45 | stdout | 2024-06-10 14:30:45 INFO     Platform: Linux-5.10.55+-x86_64-with-glibc2.31
2024/06/10 22:30:45 | stdout | 2024-06-10 14:30:45 INFO     Testing web browser installation...
2024/06/10 22:30:45 | stdout | 2024-06-10 14:30:45 INFO     FlareSolverr 3.3.19
2024/06/10 22:28:57 | stdout | 2024-06-10 14:28:57 INFO     Serving on http://0.0.0.0:8191
2024/06/10 22:28:57 | stdout | 2024-06-10 14:28:57 INFO     Test successful!
2024/06/10 22:28:57 | stdout | 2024-06-10 14:28:57 INFO     FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36
2024/06/10 22:28:56 | stdout | version_main cannot be converted to an integer
2024/06/10 22:28:56 | stdout | 2024-06-10 14:28:56 INFO     Launching web browser...
2024/06/10 22:28:56 | stdout | 2024-06-10 14:28:56 INFO     Chrome / Chromium major version: 120
2024/06/10 22:28:56 | stdout | 2024-06-10 14:28:56 INFO     Chrome / Chromium path: /usr/bin/chromium
2024/06/10 22:28:56 | stdout | 2024-06-10 14:28:56 INFO     Platform: Linux-5.10.55+-x86_64-with-glibc2.31
2024/06/10 22:28:56 | stdout | 2024-06-10 14:28:56 INFO     Testing web browser installation...
2024/06/10 22:28:56 | stdout | 2024-06-10 14:28:56 INFO     FlareSolverr 3.3.19


</body>
</html>

**Comment by hoong7** [comment]
Updated but still unsuccessful, resubmitted






---- Replied Message ----
| From | ***@***.***> |
| Date | 6/10/2024 22:09 |
| To | ***@***.***> |
| Cc | ***@***.***>,
***@***.***> |
| Subject | Re: [FlareSolverr/FlareSolverr] [chdbits] (updating) FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Error: Unable to process browser request. TimeoutError: Navigation timeout of 25000 ms exceeded (Issue #1210) |

Closed #1210 as not planned.

**Comment by ilike2burnthing** [comment]
Increase the timeout. Failing that, #1036
